test/suites/remote.sh

```diff
@@ -17,7 +17,6 @@ test_remote_url() {
 
   for url in ${urls}; do
     lxc_remote remote add test "${url}"
-    lxc_remote finger test:
```

Comment:
It's because you can't finger (GetServer()) a non-LXD server. This was accidentally working with the old code.